src/po/uk.cp1251.po

```diff
@@ -12,15 +12,15 @@
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: vim 8.0\n"
+"Project-Id-Version: vim 8.1\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2017-07-12 12:22+0300\n"
+"POT-Creation-Date: 2018-06-20 06:30+0300\n"
 "PO-Revision-Date: 2010-06-18 21:53+0300\n"
 "Last-Translator: ������� ����� <sakhnik@gmail.com>\n"
-"Language-Team: Bohdan Vlasyuk <bohdan@vstu.edu.ua>\n"
+"Language-Team: Ukrainian\n"
 "Language: uk\n"
 "MIME-Version: 1.0\n"
-"Content-Type: text/plain; charset=cp1251\n"
+"Content-Type: text/plain; charset=UTF-8\n"
```

Comment:
Is this change correct?
The filename is `uk.cp1251.po`.

Reply:
Thanks for pointing out!